source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+
+A **[scene](https://www.home-assistant.io/docs/scene/)** is a list of entities along with the state you want them each to be in. When you trigger that scene, Home Assistant will figure out the right events to get each of those entities into that state.
+
+For example, let’s imagine you have a scene called “welcome home” that says “lights on, door unlocked”. If you turn on that scene, Home Assistant will check if your lights are on and turn them on if they’re off. Then it will check if your door is unlocked, and unlock it if it’s locked.
+
+## More concepts
+
+- **Zones** are geographical areas on earth. You can check if a device tracker is inside a zone as a trigger or condition in an automation. For example, “turn on the lights when Sam’s Phone comes into the ‘home’ zone”.
+
+## Integrations
+
+Home Assistant is set up to work with devices and services from lots of different companies, including voice assistants, smart devices, cameras, and other home automation software. You can make Home Assistant work with these services by installing **[integrations](https://www.home-assistant.io/integrations)**.
+
+Once installed, integrations can make new entities available, add panels to Lovelace, and more.
+
+_Formerly known as components; not to be confused with add-ons._
+
+## The Home Assistant software
+
+So now you know what Home Assistant is. But how do you start using it?
+
+You probably want to [install Home Assistant](https://www.home-assistant.io/getting-started/) using a disk image, which contains everything you need to run Home Assistant and keep it updated.
+
+If you’re feeling adventurous, you can also install parts of Home Assistant separately:
+
+- **Core** - the main "engine" of Home Assistant that handles reading data, controlling devices, and running automations
+- **Supervisor** makes sure everything is up to date, installs add-ons, runs backups, and more
+
+Learn more about [different installation methods here](https://community.home-assistant.io/t/home-assistant-installation-methods/207703).
+
+Most people control Home Assistant with its built-in frontend, **Lovelace**. Lovelace is the user interface in your browser and the mobile apps.
+
+Lovelace is organized like this:
+
+- **Panels** are top-level pages in Lovelace. Examples are your Dashboard, the Map, the History panel and the Configuration page.
+- **[Dashboard](https://www.home-assistant.io/lovelace/dashboards-and-views)** panels are the main way to see and control your home. You can add multiple custom dashboards in addition to the one that comes by default.
+- **[Views](https://www.home-assistant.io/lovelace/dashboards-and-views)** are just tabs inside of dashboards. You can easily toggle between views on the same dashboard. You can customize the name and an icon for each view.
+- **Cards** are smaller pieces of UI inside a view. You can generally see multiple cards at once. Home Assistant comes with built in cards, but you can also write your own.
```

Comment:
Why is this at the bottom all of a sudden?

Reply:
I'm going back and forth on the flow. Where would you like it to live?

Reply:
I dunno, I just feel like this is pasted in at the end and thus a bit out of place. (disconnected from the story).